src/reference/permissions.md

```diff
@@ -15,7 +15,6 @@ The following permission tokens are pre-configured in Iroha 2:
 | [`CanRemoveKeyValueInUserAssets`](#canremovekeyvalueinuserassets)                           | Asset            | Remove key value |
 | [`CanTransferUserAssets`](#cantransferuserassets)                                           | Asset            | Transfer         |
 | [`CanTransferOnlyFixedNumberOfTimesPerPeriod`](#cantransferonlyfixednumberoftimesperperiod) | Asset            | Transfer         |
-| [`CanMintUserAssetDefinitions`](#canmintuserassetdefinitions)                               | Asset Definition | Mint             |
```

Comment:
please check my comment in the issue. It should be replaced with new tokens

Reply:
@mversic could you please assign it on me. I've started actualizing this.